app/src/ccc36c3/res/values-sv/strings.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">36C3 Schedule</string>
+    <string name="conference_url">
+        <![CDATA[<a href="https://events.ccc.de/congress/2019/wiki">https://events.ccc.de/congress/2019/wiki</a>]]>
+    "</string>
+    <string name="app_hardcoded_subtitle">December 27.-30. 2019, Leipzig, Leipziger Messe</string>
```

Comment:
Is `December 27.-30. 2019` the typical way a date is noted in Sweden?

Reply:
Ohh, no, it should be 27-30 december 2019. Will fix that one and the other strings.